api/src/data/migrations/016.create-incidents.ts

```diff
@@ -0,0 +1,29 @@
+import * as knex from "knex";
+
+export async function up(knex: knex.Knex) {
+    await knex.schema.createTable("incidents", function(table) {
+        table.increments("id").primary().notNullable();
+        table.string("reporting_person", 256).nullable();
+        table.string("proxy_employee_id", 256).nullable();
+        table.integer("proxy_role_id").nullable();
+        table.string("description", 4096).notNullable();
+        table.string("sensitivity", 8).notNullable();
+        table.datetime("created_at").notNullable().defaultTo(knex.fn.now());
+        table.string("status", 8).notNullable();
+        table.string("department", 8).notNullable();
+        table.string("supervisor", 256).nullable();
+        table.integer("incident_type_id").notNullable();
+
+        table.foreign("sensitivity").references("sensitivities.code");
+        // status foreign keys
+        // Why do we have multiple foriegn keys for status? Remove this comment once resolved
+        table.foreign("status", "incident_incident_status_code_fk").references("incident_statuses.code");
+        table.foreign("status", "incident_incident_status_code_fk2").references("incident_statuses.code");
```

Comment:
Only keep the first one, and drop the custom name.
e.g.
```suggestion
        table.foreign("status").references("incident_statuses.code");
```
also make that link to the `incident_statuses.id` column when it exists.

Reply:
Also rename `status` to `status_id`.
The pattern is `<deduplicated_external_table>_<external_column>

So normally here it would be
`incident_status_xxxx` but this is the `incident` table so you can just write `status_xxxx`.

If you didn't add an id column and link to it, this column would instead be named `incident_code` to follow the standard.